lib/pr_checker.js

```diff
@@ -51,7 +55,8 @@ class PRChecker {
     if (this.authorIsNew()) {
       status.push(this.checkAuthor());
     }
-    // TODO: maybe invalidate review after new commits?
+
+    this.checkCommitsAfterReview();
```

Comment:
Currently this doesn't affect the `status` that checkAll returns (and also has no status of its own). I also think it should very likely be checked right after `checkReviews` rather than after all those other ones... but maybe I'm wrong.